root/user/profile.tt

```diff
@@ -91,6 +91,12 @@
             [% END %]
         [% END %]
 
+        [% IF c.user_exists AND (viewing_own_profile OR c.user.is_account_admin);
+             WRAPPER property name=l('Last password login:');
```

Comment:
I think the idea is to distinguish between logging in via typing your password, and logging in via an existing session cookie, but I don't have any suggestion for a better wording. :)